sgpt/app.py

```diff
@@ -49,7 +50,9 @@ def get_completion(
 
 
 def main(
-    prompt: str = typer.Argument(None, show_default=False, help="The prompt to generate completions for."),
+    prompt: str | None = typer.Argument(
```

Comment:
Curiously, this breaks for me:

```
RuntimeError: Type not yet supported: str | None
```

I have Python 3.10.9

Reply:
Using the `typing.Optional` hint appears to be the correct way to do this.